### ISSUE TITLE
Make admin interface

### ISSUE THREAD

**Issue by rosemarybeetle** [rosemarybeetle/cloudenstein#6]
To store search variables currently in Google spreadsheet

**Comment by rosemarybeetle** [comment]
starting in models.py with
<code>search_term=models.CharField(max_length=20) # what the api is searching for

<code>tweet_num=models.CharField(max_length=3) # max value = 100

<code>harvest_period=models.CharField(max_length=60) # in seconds

<code>intro_text=models.CharField(max_length=300)

<code>sub_text=models.CharField(max_length=300)

**Comment by rosemarybeetle** [comment]
Completed - used 
<code>$ heroku run python manage.py createsuperuser<code>

**Comment by rosemarybeetle** [comment]
and created a new page to test results (craply)
https://immense-bayou-1102.herokuapp.com/tweetadmin

Finally worked out what was going on with updating too.

I was initially getting INSERT rows to work using Django save on a model instance. - SUCCESS

Trying to get an update I was initially trying to force it by always using the same value a special field, but it was not the primary key - FAIL

I then tried force_update=True using the primary key but due to table rebuilds and model changes, I was attemtping a force update separately on two rows using the primary key field (id=0 and then id=17, both of which turned out did not exist. Ironically they would have been created if I had NOT used force_update=True. 

Once I used an id that represented a real row - SUCCESS!
